content/en/gitops.md

```diff
@@ -0,0 +1,40 @@
+---
+title: GitOps
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+
+GitOps is a set of best practices based on [shared principles](https://opengitops.dev/) applied to managing and operating a cloud-native infrastructure and applications. 
+The GitOps approach is used to strategize a cohesive workflow that leverages Git as the "single source of truth" and extends this practice to an application's configuration, infrastructure, and operational procedures. 
+The GitOps controller automates matching the Git state with the system state in a bidirectional manner.
```

Comment:
Instead of `controller` possibly the better work is `agent`? or `operator`? 

Ref: https://www.weave.works/technologies/gitops/

Reply:
@ChrisJBurns - Hm, so something like "GitOps depends on software agents that automate matching the Git state with a system state in a bidirectional matter."? Because it wouldn't be "GitOps agent" or "GitOps operator".

Reply:
How about something like:

_"GitOps depends on software agents that aim to automatically reconcile declared state in Git with the realised state in system infrastructure"_ ?

Reply:
@ChrisJBurns - I like that! I'll submit that change

Reply:
@ChrisJBurns - I ended up trying to simplify this as much as I could and make it more general. When I shared the term "agent" or "controller" with non-technical people they were unsure what that was. I included tools instead, but curious what you and @JasonMorgan think

Reply:
I'm not sure using the word `tools` would be a good fit here. Tools are used to complete a task whereas Agents do the task for you - following [this article](https://medium.com/@dieswaytoofast/ftools-vs-agents-know-your-product-edition-2a2d284b3263). Although I understand it's a tricky one for non-technical folk, we want to be careful when we avoid the use of technical terms for non-technical folk benefit, that we don't actually use words in replacement that mislead the definition itself.

Reply:
@ChrisJBurns - This is a great point, "Tools are used to complete a task whereas Agents do the task for you". I also really enjoyed [MIT's](https://www.media.mit.edu/groups/software-agents/overview/) definition and will try to think of a simple way to describe this, otherwise, I'll default to "agent".

Reply:
This was updated!